### DISCUSSION TITLE
Disappearing characters when converting from pdf to tiff

### DISCUSSION THREAD

**Discussion by MrZakaz** [ImageMagick/ImageMagick#3292] [Help]
Hello
I come with a question, I used Image Magick a long time ago but when converting from PDF to Tiff it removed selected characters from text.
I would like to know if the latest versions have this error as I would like to come back to using it.

**Comment by snibgo** [comment]
If you can supply inputs that had that problem, we can test them on recent versions of IM.

**Reply by MrZakaz** [reply]
The problem is, I don't have the code ... so if it were possible, would you be able to present me a pdf to tiff converter script. then I will see that it works and that is enough for me.

**Comment by snibgo** [comment]
To convert an entire PDF:
```
magick in.pdf out.tiff
```

To convert just one page of a PDF:
```
magick in.pdf[0] out.tiff
```
Use any page number instead of 0.

I suggest you ensure you have a recent version of IM and Ghostscript, which does the rasterization.

**Reply by MrZakaz** [reply]
Would you be able to send me a file and its result after conversion? Preferably with a lot of text.
I will be very grateful

**Comment by snibgo** [comment]
Why? What would that show? Many PDFs are available on the internet. You can convert them for yourself.

**Reply by MrZakaz** [reply]
I'd like to make sure it works, and I don't have an environment to test myself.
